core/src/main/scala/cats/arrow/Compose.scala

```diff
@@ -7,8 +7,11 @@ import simulacrum.typeclass
  * Must obey the laws defined in cats.laws.ComposeLaws.
  */
 @typeclass trait Compose[F[_, _]] { self =>
+
+  @simulacrum.op(":<<", alias = true)
```

Comment:
I don't like the asymmetry here. If we have the `:<<` The other should be `>>:`.